src/apps/users/services/core.py

```diff
@@ -66,8 +74,18 @@ async def send_password_recovery(self, schema: PasswordRecoveryRequest) -> Publi
 
         exp = settings.authentication.password_recover.expiration // 60
 
+        # Default to web frontend
+        frontend_base_url = settings.service.urls.frontend.web_base
+        if content_source == MindloggerContentSource.admin:
+            # Change to admin frontend if the request came from there
+            frontend_base_url = settings.service.urls.frontend.admin_base
+
+        base_url_with_protocol = frontend_base_url
+        if not base_url_with_protocol.startswith("http"):
```

Comment:
Both settings web_base and admin_base are only domains, so do not need to check if it starts with http

Reply:
That only applies to the static values in the `FrontendUrlsSettings`

https://github.com/ChildMindInstitute/mindlogger-backend-refactor/blob/7a5626be83abbac4d05d2bf303b6f2a2ac031db0/src/config/service.py#L6-L8

However, for local testing I override these values with http URLs

```shell
SERVICE__URLS__FRONTEND__WEB_BASE="http://localhost:5173"
SERVICE__URLS__FRONTEND__ADMIN_BASE="http://localhost:3000"
```

So this change was necessary to account for that

Reply:
but this is only in local development, do not have protocol in any other environment. you can use ngrok or any other tunnel service to have https

Reply:
Well you're right. It doesn't affect the other environments, so I thought it would be innocuous enough. I'll make the change